build-info-extractor-pip/src/main/java/org/jfrog/build/extractor/pip/extractor/DependenciesCache.java

```diff
@@ -0,0 +1,108 @@
+package org.jfrog.build.extractor.pip.extractor;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import org.jfrog.build.api.Dependency;
+import org.jfrog.build.api.util.Log;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Bar Belity on 19/07/2020.
+ */
+public class DependenciesCache {
```

Comment:
Let's add a description for this class, explaining the function of this class.

Reply:
Added.